README.md

```diff
@@ -2,10 +2,100 @@
 
 Requires dbt >= 0.12.2
 
+### about dbt
+
 ---
```

Comment:
I'm very happy to delete this section! It's auto-generated from when we create a package :(

Reply:
Sounds good, I'll add a link to the DBT homepage